lib/jobs/isc-dhcp-lease-poller.js

```diff
@@ -144,6 +144,10 @@ function iscDhcpLeasePollerFactory(
         */
         var split = data.split('\n');
         var lease;
+        var todaysDate = new Date();
+        var notExpired = false;
```

Comment:
This double negative is also kind of confusing, can we just do: `var expired = true`?